rollup.config.js

```diff
@@ -47,5 +43,9 @@ export default {
 			include: 'node_modules/**',
 			exclude: '**/*.css'
 		})
+	],
+	targets: [
+		{ dest: pkg.main, sourceMap: path.resolve(pkg.main), format: 'umd', moduleName: pkg.amdName },
```

Comment:
don't know, the original config had that `path.resolve`. Let me try what happens with `sourceMap: true`